lib/formatter/csv/user_email.rb

```diff
@@ -0,0 +1,17 @@
+module Formatter
+  module Csv
+    class UserEmail
+      def headers
+        false
+      end
+
+      def to_rows(user)
+        [to_array(user)]
+      end
+
+      def to_array(user)
```

Comment:
seems like overkill

Reply:
Ah, yeah with the addition of `to_rows` to all formatters (to maintain the same interface between everything including subjects) this could just be inlined into `to_rows`.

`to_array` should just be a private method anyway.